src/mrpro/data/AcqInfo.py

```diff
@@ -251,17 +270,37 @@ def spatialdimension_2d(
             flags=tensor_2d(headers['flags']),
             measurement_uid=tensor_2d(headers['measurement_uid']),
             number_of_samples=tensor_2d(headers['number_of_samples']),
+            orientation=orientation_from_directional_cosine(
+                spatialdimension_2d(headers['slice_dir']),
```

Comment:
Suggestion: move the apply of conversion to here 
Such that SpatialDimension2d only creates a 2d SpatialDimension,  and the conversion is done explicitly in each line. Makes it easier to see what is happening

Reply:
I can't comment on the SpatialDimension2d code where it should be changed..